Apsitvarkom.Api/Controllers/CleaningEventController.cs

```diff
@@ -40,7 +40,9 @@ public async Task<ActionResult<IEnumerable<CleaningEventResponse>>> GetAll()
             return StatusCode(StatusCodes.Status500InternalServerError);
         }
 
-        var mappedEvents = _mapper.Map<IEnumerable<CleaningEventResponse>>(events);
+        IEnumerable<CleaningEvent> sortedEvents = events.OrderByDescending(o => o.StartTime);
```

Comment:
we can simply use [`var`](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables) here instead of explicitly specifying the type.